__tests__/NCN.test.js

```diff
@@ -303,12 +303,72 @@ describe('POST', () => {
     });
 });
 
-xdescribe('PATCH', () => {
+describe('PATCH', () => {
     describe('PATCH /api/articles/:article_id', () => {
         test('PATCH: 200 update an article by ID and responds with updated article', () => {
             const postObj = {inc_votes: 20}
             return request(app)
+            .patch('/api/articles/1')
+            .send(postObj)
+            .expect(200)
+            .then((response) => {
+                const updatedArticle = response.body
```

Comment:
again we haven't made use of the updatedArticle here to actually test that it is an object containing the below